meta-facebook/yv4-sd/src/platform/plat_isr.c

```diff
@@ -136,6 +136,12 @@ add_sel_info event_work_items[] = {
 		.event_type = P12V_STBY_UV,
 		.assert_type = EVENT_ASSERTED,
 	},
+	{
+		.is_init = false,
+		.gpio_num = CPU_SMERR_BIC_N,
+		.event_type = SYS_MANAMENT_ERROR,
```

Comment:
@RushChen1 is this a typo? MANAMENT
I suspect you meant MANAGEMENT. Wonder how its compiling.

Reply:
ok, thank you!